### ISSUE TITLE
Not able to connect to myQ

### ISSUE THREAD

**Issue by dvalenzuela** [hjdhjd/homebridge-myq#868]
**Describe The Problem:**
When trying to use the plugin through hombridge, I'm getting 
[myQ] myQ API: Invalid myQ credentials given. Check your login and password.

**To Reproduce:**
Opened the app and now its not working.

**Logs:**

```
 [9/29/2022, 5:05:40 PM] Homebridge v1.4.1 (HAP v0.10.2) (Homebridge 9A45) is running on port 51301.
[9/29/2022, 5:05:41 PM] [myQ] myQ API: https://partner-identity.myq-cloud.com/connect/authorize?client_id=IOS_CGI_MYQ&code_challenge=L24J6P4rrU5QI82FWNNs0SjYFISM1dqTsTbCYiIK2bc&code_challenge_method=S256&redirect_uri=com.myqops%3A%2F%2Fios&response_type=code&scope=MyQ_Residential+offline_access Error: 502 Bad Gateway
[9/29/2022, 5:05:41 PM] [myQ] myQ API: Unable to access the OAuth authorization endpoint.
[9/29/2022, 5:28:19 PM] TypeError: Cannot read property 'map' of undefined
    at myQApi.trimSetCookie (/Users/admin/.nvm/versions/node/v15.14.0/lib/node_modules/homebridge-myq/node_modules/@hjdhjd/myq/src/myq-api.ts:688:22)
    at myQApi.oauthLogin (/Users/admin/.nvm/versions/node/v15.14.0/lib/node_modules/homebridge-myq/node_modules/@hjdhjd/myq/src/myq-api.ts:150:25)
    at myQApi.getOAuthToken (/Users/admin/.nvm/versions/node/v15.14.0/lib/node_modules/homebridge-myq/node_modules/@hjdhjd/myq/src/myq-api.ts:234:27)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
    at myQApi.acquireAccessToken (/Users/admin/.nvm/versions/node/v15.14.0/lib/node_modules/homebridge-myq/node_modules/@hjdhjd/myq/src/myq-api.ts:364:19)
    at myQApi.refreshAccessToken (/Users/admin/.nvm/versions/node/v15.14.0/lib/node_modules/homebridge-myq/node_modules/@hjdhjd/myq/src/myq-api.ts:406:14)
    at myQApi.refreshDevices (/Users/admin/.nvm/versions/node/v15.14.0/lib/node_modules/homebridge-myq/node_modules/@hjdhjd/myq/src/myq-api.ts:448:10)
    at myQPlatform.updateAccessories (/Users/admin/.nvm/versions/node/v15.14.0/lib/node_modules/homebridge-myq/src/myq-platform.ts:285:10)
    at /Users/admin/.nvm/versions/node/v15.14.0/lib/node_modules/homebridge-myq/src/myq-platform.ts:326:14
```

**Homebridge Configuration:**

=
**Environment:**

* **Homebridge Version:** 1.4.1
* **Node Version:**v15.14.0
* **Homebridge-myQ Plugin Version**:homebridge-myq v2.7.4
* **Mac
* **MacOs

**Comment by thabandit007** [comment]
Same issue as well, seems like API has been changed:
myQ API: Unable to access the OAuth authorization endpoint.

**Comment by sifield** [comment]
Same here, stopped working yesterday: myQ API: Unable to access the OAuth authorization endpoint.

**Comment by DMBlakeley** [comment]
I also noticed that the plugin was not working last evening. I also noticed that the MyQ app was not working. Going to [downdetector.com](downdetector.com) that there were issues with the MyQ servers. Once this cleared up MyQ and the plugin starting working again. Today, no issue.

I do not believe that there is an issue with the plugin. If you are still experiencing an issue I would recommend restarting Homebridge to reinitialize connection with MyQ.
